.golangci.yml

```diff
@@ -87,3 +87,7 @@ issues:
     # Duplicate words are okay in tests.
     - linters: [dupword]
       path: _test\.go
+
+    - linters: [maintidx]
+      path: context\.go
+      text: 'Function name: trace'
```

Comment:
To bypass the linter error:
```
context.go:350:1: Function name: trace, Cyclomatic Complexity: 47, Halstead Volume: 4627.00, Maintainability Index: 18 (maintidx)
```

Would you prefer I extracted something from `trace` instead? Or reworked it to not have a nested `if`?

... maybe like
```golang
case FlagToken:
	if err := c.parseFlag(flags, token.String()); err == nil {
		// Flag was parsed successfully.
	} else if isUnknownFlagError(err) && positional < len(node.Positional) && node.Positional[positional].Passthrough {
		c.scan.Pop()
		c.scan.PushTyped(token.String(), PositionalArgumentToken)
	} else {
		return err
	}
```

Reply:
It's fine, the linters are just guides, I usually disable them as necessary. FYI you can also just use `//nolint:maintidx` on the offending line.